File: public/phase-2-20160620022203/PYTHON/boy-and-boxes.py
#!/usr/bin/python
# Name : Boy and Boxes
# input:
# n :  the number of boxes
# x : number of the
# a :  number of balls in each box
elements = raw_input().strip().split(' ')
n = int(elements[0])
x = int(elements[1])
a_elements = raw_input().strip().split(' ')
a = map(int, a_elements)


# write your code here
# store your results in `result`

# output

# Dummy Data
result = [3,2,5,4]

print(" ".join(str(e) for e in result))

File: public/phase-2-20160620022203/C/boy-and-boxes.c
// Name : Boy and Boxes
#include <stdio.h>
int main() {
	// input:
	// n :  the number of boxes
	// x : number of the
	// a :  number of balls in each box
	int n, x, a[10000], result[10000]={3,2,5,4}, index;
	scanf("%d %d", &n, &x);
	for(index = 0; index< n; index++) {
		scanf("%d", &a[index]);
	}


	// write your code here
	// store your results in `result`

	// output
	for(index = 0; index< n; index++) {
		printf("%d ",result[index]);
	}
	return 0;
}